documentation/src/main/wiki/Splitter.md

```diff
@@ -4,14 +4,14 @@ goes to Knot context and can be processed later by [[Knots|Knot]].
 
 ##How does it work?
 HTML Fragment Splitter gets a Knot context as input and responds with the modified Knot context. It 
-divides HTML using regexp `<script\s+data-api-type\s*=\s*"([A-Za-z0-9-]+)"[^>]*>.+?</script>` into
-static and dynamic fragments. So all `script` tags with a `data-api-type` attribute are converted to 
-dynamic fragments. **According to performance reasons Splitter requires `data-api-type` 
+divides HTML using regexp `<script\s+data-knot-types\s*=\s*"([A-Za-z0-9-]+)"[^>]*>.+?</script>` into
+static and dynamic fragments. So all `script` tags with a `data-knot-types` attribute are converted to 
+dynamic fragments. **According to performance reasons Splitter requires `data-knot-types` 
 attribute to be the first attribute in the `script` tag.**
 All HTML markup outside script tags is considered as static fragments.
 
-Fragment contains an *identifier, a content (a template chunk) and a context*. The *identifier* has `data-api-type` 
-attribute value or `_raw` for static fragments. It can be used by Knots to select required fragment / fragments 
+Fragment contains an *list of knot types, a content (a template chunk) and a context*. The *list of knot types* contains `data-knot-types` 
```

Comment:
This whole paragraph seems to me as something to rewrite - I don't understand what it tires to explain..
what do you mean by *context* ???
Are you trying to describe what is the fragment from Knot Context point of view or in general ?
Instead simply put explanations that *data-knot-types* is used for - to say which knots should process the snippet.